.gitignore

```diff
@@ -36,3 +36,5 @@ coverage
 .env
 .env.development
 .env.production
+
+config/initializers/explain_partials.rb
```

Comment:
Why aren't we adding explain_partials.rb to the codebase?  I'd love it to be there in all our projects and just have a test never to run in the rails `production` environment.